sdks/nodejs/src/agonesSDK.js

```diff
@@ -23,7 +23,20 @@ class AgonesSDK {
 		this.healthStream = undefined;
 		this.emitters = [];
 	}
-	async close(){
+
+	async connect() {
```

Comment:
We should add a test for this to maintain code coverage. Please let me know if you'd like me to add, e.g. separate PR?

Reply:
Not sure how we are going to make a unit test for this, when there is nothing to connect to, but I will have a look, see if I can work out a way.

Reply:
Aha, got it - there is a mocking library you use to mock out the client. On it!

Reply:
Done! PTAL.